### ISSUE TITLE
Module is having compilation issues with the new release of Xcode 10.2 (10E125) 

### ISSUE THREAD

**Issue by piemonte** [Svrf/svrf-swift4-client#4]
```sh
The target “SVRFClient” contains source code developed with Swift 3.x. This version of Xcode does not support building or migrating Swift 3.x targets.

Use Xcode 10.1 to migrate the code to Swift 4.
```

**Comment by piemonte** [comment]
Repro steps:

Update to Xcode 10.2 (10E125)
Create a fresh Xcode project
pod init
Add pod Apollo to the Podfile
pod install
See that cocoapods installed SVRFClient 1.4.2
Open the .xcworkspace
Attempt to compile the project
See the compiler error

**Comment by brentchow** [comment]
closed by #5. Released in [1.6.0](https://github.com/Svrf/svrf-swift4-client/releases/tag/1.6.0)
